redpanda_build/CMakeLists.txt

```diff
@@ -48,3 +48,6 @@ target_include_directories(avro
   PRIVATE ../lang/c++/include/avro)
 
 add_library(Avro::avro ALIAS avro)
+
+add_executable (avrogencpp impl/avrogencpp.cc)
```

Comment:
i didn't mean literally copy the lines :) notice above that the paths are relative to this file ` ../lang/c++/impl/CustomAttributes.cc)`.